albumentations/augmentations/transforms.py

```diff
@@ -3124,9 +3165,7 @@ class FancyPCA(ImageOnlyTransform):
     """
 
     def __init__(self, alpha=0.1, always_apply=False, p=0.5):
-        """
-
-        """
+        """"""
```

Comment:
Looks like this row could be safely deleted since it is an empty docstring.

Reply:
Done!